src/Search/Search.js

```diff
@@ -32,22 +33,22 @@ function WikipediaSearch() {
       srsearch: searchTerm,
     };
   
-    await fetch(url + new URLSearchParams(params).toString())
-      .then((response) => {
-        if (!response.ok) {
-          throw new Error('Network response was not ok');
-        }
-        return response.json();
-      })
-      .then((data) => {
+  await fetch(url + new URLSearchParams(params).toString())
+    .then((response) => {
+      if (!response.ok) {
+        throw new Error('Network response was not ok');
+      }
+      return response.json();
+    })
+    .then((data) => {
 // Sets the state of the initial results to the first result of the search (i.e. Rush Limbaugh)
-        setInitialResults(data.query.search[0]);
-        navigate(`/article/${data.query.search[0].pageid}`);
-      })
-      .catch((error) => {
-        console.error('There was a problem with the fetch operation:', error);
-      });
-  }
+      setInitialResults(data.query.search[0]);
```

Comment:
`data.query.search[0]` is quite a handful - could you store this in a variable with an appropriate variable name to improve readability?